src/dfx/src/lib/operations/canister/install_canister.rs

```diff
@@ -76,15 +76,11 @@ pub async fn install_canister(
                 Ok(None) => (),
                 Ok(Some(err)) => {
                     let msg = format!("Candid interface compatibility check failed for canister '{}'.\nYou are making a BREAKING change. Other canisters or frontend clients relying on your canister may stop working.\n\n", canister_info.get_name()) + &err;
-                    ask_for_consent(&msg).map_err(|e| {
-                        anyhow!("Refusing to install canister without approval: {e}")
-                    })?;
```

Comment:
this is intentional and ok, see https://github.com/dfinity/sdk/blob/master/src/dfx-core/src/error/canister.rs#L14